Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
>       print(__RomanNumbers._RomanNumbers__encode_digit(5, 0))

../test.py:5: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'string_utils.manipulation.__RomanNumbers'>, index = 5, value = 0

    @classmethod
    def __encode_digit(cls, index: int, value: int) -> str:
        # if digit is zero, there is no sign to display
        if value == -1:
            return ''
    
        # from 1 to 3 we have just to repeat the sign N times (eg: III, XXX...)
        if value <= 3:
>           return cls.__mappings[index][1] * value
E           IndexError: list index out of range

string_utils/manipulation.py:60: IndexError
=========================== short test summary info ============================
FAILED ../test.py::test - IndexError: list index out of range
============================== 1 failed in 0.05s ===============================
